packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -493,6 +493,23 @@ function doMovePoint(
                 return state;
             }
 
+            // If the last point is being moved to the same position as
+            // the first point, close the polygon.
+            if (
```

Comment:
I think I ended up removing this code because this function should only be marking the polygon as closed? I noticed in the tests that the polygon's state changed to close even if it had an empty coords array, so I removed these lines to stay consistent with that behavior.